lib/awful/hotkeys_popup/widget.lua

```diff
@@ -311,14 +311,14 @@ function widget.new(args)
     end
 
 
-    function widget_instance:_abbreviate_hotkeys(target)
+    function widget_instance:_abbreviate_hotkeys()
```

Comment:
as its sorting only `awful` keys now i'd rename it into smth like _abbreviate_awful_keys()

Reply:
Okay, done.